README.md

```diff
@@ -138,6 +138,35 @@ As soon as that line of code is reached in any of the running instances of your
 
 ![Breakpoint Hit](doc/images/breakpoint-hit.png?raw=true)
 
+Note that the directory layout of the code that is being debugged does not have to exactly match the source code specified in the Debug UI.  In particular, if a snapshot is specified in a file in the Debug UI, a breakpoint is set in the deployed file with the longest matching path suffix.
```

Comment:
copy editing nits: 

* highlight *not* in first sentence.
* second sentence would read better like this: "The agent resolves snapshot filenames by searching for a file with the longest matching path suffix. If a use unique match is found, we will use that to set the snapshot.